packages/jest-runner/src/runTest.ts

```diff
@@ -182,12 +191,14 @@ async function runTestInternal(
 
   if (
     environment.global &&
-    environment.global.process &&
-    environment.global.process.exit
+    (environment.global as NodeJS.Global).process &&
```

Comment:
Not sure about this. I checked out the types in `Environment.$JestEnvironment` and it overriding `NodeJS.global`

Reply:
I corrected the type, so the cast isn't necessary anymore